sdkconfig

```diff
@@ -206,6 +206,12 @@ CONFIG_EFUSE_MAX_BLK_LEN=192
 # ESP32-specific
 #
 CONFIG_IDF_TARGET_ESP32=y
+CONFIG_ESP32_REV_MIN_0=y
```

Comment:
Which IDF are you using? If it's an IDF update, it should be an independent commit.

Reply:
Are these all the default values?

Reply:
no removed and enabled by adding the entries on the kconfig on the idf folder (modifying the kconfig) say me if wrong procedure